asciidoctorj-cli/src/main/java/org/asciidoctor/cli/jruby/AsciidoctorInvoker.java

```diff
@@ -1,29 +1,32 @@
 package org.asciidoctor.cli.jruby;
 
 import com.beust.jcommander.JCommander;
-import org.asciidoctor.Asciidoctor;
-import org.asciidoctor.Options;
 import org.asciidoctor.cli.AsciidoctorCliOptions;
 import org.asciidoctor.cli.MaxSeverityLogHandler;
 import org.asciidoctor.jruby.DirectoryWalker;
 import org.asciidoctor.jruby.GlobDirectoryWalker;
 import org.asciidoctor.jruby.internal.IOUtils;
 import org.asciidoctor.jruby.internal.JRubyAsciidoctor;
 import org.asciidoctor.jruby.internal.JRubyRuntimeContext;
+import org.asciidoctor.jruby.internal.RubyGemsPreloader;
 import org.asciidoctor.jruby.internal.RubyUtils;
 import org.jruby.Main;
+import org.jruby.Ruby;
+import org.jruby.RubyClass;
+import org.jruby.RubyHash;
 import org.jruby.runtime.builtin.IRubyObject;
 
 import java.io.File;
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.URL;
 import java.net.URLClassLoader;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.Properties;
 import java.util.stream.Collectors;
 
-import static org.asciidoctor.cli.AsciidoctorCliOptions.TIMINGS_OPTION_NAME;
-
 public class AsciidoctorInvoker {
```

Comment:
IDE flags some unnecessary `throws` and methods, one is concerning `readInputFromStdIn`. Is passing console input still possible?

Reply:
> Is passing console input still possible?

Yes, if Asciidoctor Ruby supports it, AsciidoctorJ will as well. 
File or stream handling is completely passed to Asciidoctor Ruby, except for finding all input file names when glob expressions are used. (I would love to drop that as well, the current architecture doesn't allow for that though)